README.md

```diff
@@ -129,7 +129,7 @@ A client for accessing the Smartcar API
 | `client_secret` | String |**Required** Application clientSecret obtained from [Smartcar Developer Portal](https://developer.smartcar.com). If you do not have access to the dashboard, please [request access](https://smartcar.com/subscribe). |
 | `redirect_uri`  | String |**Required** RedirectURI set in [application settings](https://developer.smartcar.com/apps). Given URL must match URL in application settings. |
 | `scope`         | String[] |**Optional** List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs#get-all-vehicles). |
-| `development`   | Boolean |**Optional** Launch Smartcar auth in development mode to enable the mock vehicle brand. |
+| `test_mode`   | Boolean |**Optional** Launch Smartcar auth in test mode to enable the mock vehicle brand. |
```

Comment:
the last portion of this comment is incorrect, it doesn't only enable the mock vehicle brand, but rather puts all vehicles into mock mode

cc: @esonmez

Reply:
I've updated the spec to include standard copy for the `testMode` flag: https://smartcar.quip.com/CS0aAP8GEMEU#cPWACA9LYFn

Reply:
Prefer:
```
**Optional** Launch the Smartcar auth flow in test mode.
```